packages/zowe-explorer/src/trees/dataset/DatasetTree.ts

```diff
@@ -136,6 +150,7 @@ export class DatasetTree extends ZoweTreeProvider<IZoweDatasetTreeNode> implemen
             const children = await sourceNode.getChildren();
             for (const childNode of children) {
                 // move members within the folder to the destination
+                childNode.contextValue = 'member'; //force casting binary members to member context
```

Comment:
Overwriting the `contextValue` of a node could have unintended side effects, since there may be additional data stored there like encoding or profile type.

Was this change needed to work around behavior in `crossLparMove`? If so, perhaps we can update code inside that method to correctly determine whether or not a node is a member.